so_vector/operations/default.json

```diff
@@ -0,0 +1,242 @@
+{
+  "name": "knn-search-10-50-acceptedAnswerId",
+  "operation-type": "raw-request",
+  "path": "/vectors/_knn_search",
```

Comment:
if we modify the target of this PR to a new branch (`8.4`) we can just use the canonical endpoint, unless we want to be able to test < 8.4

Reply:
If it's fine with you, I'd like to move everything over to the new endpoint in a follow-up PR -- we'll have to do that for both the original `dense_vector` track and this one.

Reply:
Yes. note you can do it ahead of time by establishing an `8.4` branch, and you don't need to "time" the existence or anything. note that if you want to compare across versions you'll want to have a `name` for the tasks that you intend to keep regardless of version

Reply:
👍 that makes sense